lib/src/components/NxCodeSnippet/NxCodeSnippet.scss

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+.nx-code-snippet {
```

Comment:
I think this component is missing some margin - two of them next to each other don't have any space in between.

Reply:
> I think this component is missing some margin - two of them next to each other don't have any space in between.

Added margin matching `NxFormGroup` 31c19012